src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -1,2 +1,2 @@
 <%@tag description="Common page content" pageEncoding="UTF-8"%>
-<h3>Snippet</h3>
+<h1><span class="blue">S</span><span class="red">n</span><span class="yellow">i</span><span class="blue">p</span><span class="green">p</span><span class="red">e</span><span class="yellow">t</span></h1>
```

Comment:
only thing i'm not sure about is the format here.

Reply:
Yeah, there are not intended white spaces between the spans also when I use normal HTML format here :/